compiler/vm/vmgen.nim

```diff
@@ -913,6 +913,22 @@ proc genCall(c: var TCtx; n: PNode; dest: var TDest) =
 template isGlobal(s: PSym): bool = sfGlobal in s.flags and s.kind != skForVar
 proc isGlobal(n: PNode): bool = n.kind == nkSym and isGlobal(n.sym)
 
+template local(sym: PSym): TDest =
```

Comment:
I personally much prefer passing in all parameters instead of late binding with mixin. I imagine type might be a thing, so I tend to use a type class.

Reply:
I do too. To be honest, I have no idea why I even used a template here in the first place 😅